swarm/tools/run_inspector.py

```
@@ -33,6 +33,7 @@
 
 from swarm.config.flow_registry import get_sdlc_flow_keys  # noqa: E402
 from swarm.flowstudio.schema import StepStatusEnum  # noqa: E402
+from swarm.runtime.safe_paths import validate_path_component  # noqa: E402
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)